src/sage/rings/function_field/function_field.py

```diff
@@ -1224,6 +1227,131 @@ def completion(self, place, name=None, prec=None, gen_name=None):
         from .maps import FunctionFieldCompletion
         return FunctionFieldCompletion(self, place, name=name, prec=prec, gen_name=gen_name)
 
+    def hilbert_symbol(self, a, b, P):
+        """
+        Return the Hilbert symbol `(a,b)_{F_P}` (where `F_P` is the
+        completion of ``self`` at the place ``P``).
+
+        The Hilbert symbol `(a,b)_{F_P}` is `0` if one of ``a`` or ``b`` is
+        zero. Otherwise it takes the value `1` if  the quaternion algebra
+        defined by `(a,b)` over `F_P` is split, and `-1` if it is division.
+
+        We use the completion at the place `P` to compute the valuations `v(a)`
+        and `v(b)` as well as elements `a0` and `b0` such that, for a uniformizer
+        `pi` of the unique maximal ideal of the completion, the elememts
+        `a*pi^{-v(a))}` and `a0` respectively the elements `b*pi^{-v(b)}` and `b0`
+        are congruent modulo `pi`. Motivated by formula 12.4.10 in [Voi2021]_.
+
+        Currently only tested for function fields separable over their base
+        since places are not fully supported for other function fields. Only
+        implemented for global function fields of odd characteristic.
+
+        INPUT:
+
+        - ``a`` and ``b``: Elements of ``self``
+
+        - ``P``: A place of ``self``
+
```

Comment:
Please see the developer guide to format "INPUT:" section properly.

You may write "elements of this function field" and "a place of this function field".

Reply:
I meant that `:` should be `--`.  See here: https://doc.sagemath.org/html/en/developer/coding_basics.html#documentation-strings

Reply:
Sorry, I missed that in my code - it's fixed now.